admin/services/access-arrangements.service.js

```diff
@@ -13,7 +13,12 @@ const accessArrangementsService = {}
  * @returns {Promise<Array>}
  */
 accessArrangementsService.getAccessArrangements = async () => {
-  return accessArrangementsDataService.sqlFindAccessArrangements()
+  let aaData = await accessArrangementsDataService.sqlFindAccessArrangements()
+  const retroInputAssistantItemIndex = R.findIndex(R.propEq('code', 'RIA'))(aaData)
+  if (retroInputAssistantItemIndex > -1) {
+    aaData = R.remove(retroInputAssistantItemIndex, 1, aaData)
```

Comment:
Just a thought, if RIA was used the previous time, won't it be needed the next time, except hopefully not retrospectively?  Shouldn't RIA be converted to ITA?

Reply:
this is a distinct access arrangement, and is removed from here so it doesn't end up on the main AA page as an option for selection.  it is only ever added from the dedicated retro input assistant page.  The list above is used in the main AA page